medicines/doc-index-updater/src/create_manager/sftp_client.rs

```diff
@@ -87,6 +87,23 @@ async fn retrieve_file_from_sftp(
         Err(_) => path,
     };
     tracing::info!("{:?}", path);
+
+    // Additional logging to debug observed sftp issue
```

Comment:
Do we avoid comments where at all possible and make the code alone communicate?

Reply:
I just meant this to indicate that it's temporary, linked to debuggin this issue, so should be removed in clean-up.

Reply:
FWIW, I like comments.

Reply:
cool @TimboTambo - that's fine then